clients/ts/signalr/tests/HttpConnection.test.ts

```diff
@@ -165,8 +164,8 @@ describe("HttpConnection", () => {
             stop(): Promise<void> {
                 return Promise.resolve();
             },
-            onclose: undefined,
-            onreceive: undefined,
+            onclose: null,
+            onreceive: null,
```

Comment:
If these were `onclose?` and `onreceive?` then you would just omit them entirely.